frontend/cts/src/main/java/com/amazonaws/ml/mms/cts/Cts.java

```diff
@@ -69,14 +69,14 @@ private void startTest() {
                             "https://s3.amazonaws.com/mxnet-model-server/onnx-arcface/input2.jpg",
                             "player1.jpg");
 
-            HttpClient client = new HttpClient();
+            HttpClient client = new HttpClient(8081, 8080);
```

Comment:
What is this file? Its called Test but its under source. This should probably be moved to test folders if its a test file.

Reply:
This is not unittest and we don't want to run on every build.

Reply:
Oh. I am not sure what these files are or where they are used then. I will skip reviewing this.